Makefile

```diff
@@ -40,10 +40,29 @@ mypy:
 lint: black flake8 mypy
 
 test-unit:
-	pytest
+	pytest pyteal tests/unit
+
+sandbox-dev-up:
+	docker-compose up -d algod
+
+sandbox-dev-stop:
+	docker-compose stop algod
 
 build-and-test: check-generate-init lint test-unit
 
+# TODO: set NUM_PROCS = auto when the following issue has been fixed https://github.com/algorand/pyteal/issues/199
+NUM_PROCS = 1 
+integration-run:
+	pytest -n $(NUM_PROCS) --durations=10 -sv tests/integration
+
+integration-test: integration-run
```

Comment:
Followup #294 also runs the demo here, hence the seemingly redundant build target